### ISSUE TITLE
OWL 2 is not compatible with RDF 1.1 

### ISSUE THREAD

**Issue by sesuncedu** [owlcs/owlapi#172]
Peter @ansell points out in a comment on  #165 that RDF 1.1 changes the interpretation of literals  without explicit datatypes.

If the literal has no explicit datatype, and no language tag, instead of being a rdf:PlainLiteral with   no language, it is now an xsd:string.

If  the literal has no explicit datatype, but does have a language tag, it is now an rdf:langString. 

The [OWL2 RDF mapping](http://www.w3.org/TR/2012/REC-owl2-mapping-to-rdf-20121211/)  document  continues to state in Table 1 that

```
abc@"^^rdf:PlainLiteral ===>            "abc"
"abc@langTag"^^rdf:PlainLiteral ===>    "abc"@langTag
```

and 
The AS/FSS document states 

> Literals represent data values such as particular strings or integers. They are analogous to typed RDF literals [RDF Concepts] and can also be understood as individuals denoting data values. Each literal consists of a lexical form, which is a  string, and a datatype; the datatypes supported in OWL 2 are described in more detail in Section 4. A literal consisting of a lexical form "abc" and a datatype identified by the IRI datatypeIRI is written as "abc"^^datatypeIRI. Furthermore, literals whose datatype  is rdf:PlainLiteral can be abbreviated in functional-style syntax ontology documents as plain RDF literals [RDF Concepts]. These abbreviations are purely syntactic shortcuts and are thus not reflected in the structural specification of OWL 2. The observable behavior of OWL 2 implementation must be as if these shortcuts were expanded during parsing.

Literals of the form "abc@"^^rdf:PlainLiteral should be abbreviated in functional-style syntax ontology documents to "abc" whenever possible.
Literals of the form "abc@langTag"^^rdf:PlainLiteral where "langTag" is not empty should be abbreviated in functional-style syntax documents to "abc"@langTag whenever possible.

the rdf:PlainLiteral specification adds:

> To eliminate another source of syntactic redundancy and to retain a large degree of interoperability with applications that do not understand the rdf:PlainLiteral datatype, the form of rdf:PlainLiteral literals in syntaxes for RDF graphs and for SPARQL is the already existing syntax for the corresponding plain literal, not the syntax for a typed literal. Therefore, typed literals with rdf:PlainLiteral as the datatype are considered by this specification to be not valid in syntaxes for RDF graphs or SPARQL.

TL;DR - Doooooooooom!.
